e2e_test/tests/editor.spec.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable import/no-unresolved */
+const { test, expect } = require('@playwright/test');
+
+test('Get Main Page', async ({ page }) => {
+  await page.goto('https://da.live/');
```

Comment:
We want this on the PR URL, right?

We did this in our playwright project... we grabbed the incoming branch name and constructed the URL to run the tests.

Reply:
Of course, thanks. I've done that now, the test URL is echoed so you can verify it: https://github.com/adobe/da-live/actions/runs/8419149350/job/23051143836?pr=85#step:6:9